commands/stonks.js

```diff
@@ -1,18 +1,40 @@
-exports.run = async (client, message, args, level) => { // eslint-disable-line no-unused-vars
-  const msg = await message.channel.send("Ping?");
-  msg.edit(`Pong! Latency is ${msg.createdTimestamp - message.createdTimestamp}ms. API Latency is ${Math.round(client.ping)}ms`);
+
+const finnhub = require('finnhub');
+const config = require("../config.js");
+
+const api_key = finnhub.ApiClient.instance.authentications['api_key'];
+api_key.apiKey = config.finnhub_api_key; 
+const finnhubClient = new finnhub.DefaultApi();
+
+exports.run = async (client, message, args, level) => {
+  console.log(args);
+  const company = args[0];
+  console.log(company);
+
+  if(company) {
+    finnhubClient.quote("VEEV", (error, data, response) => {
```

Comment:
wow

Reply:
You should add a TODO/another ticket so we don't forget to not make this hardcoded

Reply:
I see you addressed this. Thank you.